newspaper/network.py

```diff
@@ -59,12 +59,8 @@ def get_html_2XX_only(url, config=None, response=None):
     if response is not None:
         return _get_html_from_response(response)
 
-    try:
-        response = requests.get(
-            url=url, **get_request_kwargs(timeout, useragent, proxies, headers))
-    except requests.exceptions.RequestException as e:
```

Comment:
Good catch, this try / catch handler is not needed since the call sites seem to all have the same handler anyways..

Reply:
However FYI that the design in the html fetch functions of catch exceptions is intentional because otherwise when we are downloading 1000+ articles from CNN for example, if 100 articles gave a timeout error it would make more sense to catch those errors and print the messages instead of throw an individual uncaught exception for the first failed article and kill the whole flow